src/Drupal/DrupalExtension/Context/RawDrupalContext.php

```diff
@@ -275,7 +275,7 @@ public function nodeCreate($node) {
 
   /**
    * Parse multi-value fields. Possible formats:
-   *    A, B, C
+   *    A, B, "C, D"
```

Comment:
How's this:
   * Parse a table row as a Drupal field, allowing for multi-value and compound fields. 
   * Compound field properties are identified using a ':' operator, either in the column
   * heading or in the cell. If multiple properties are present in a single cell, they must be
   * separated using ' - ', and values should not contain ':' or ' - '.
   * Multiple values in a single cell must be separated using ', ' and may be wrapped in
   * double quotes where necessary. 
   * Possible formats:
   * A
   * A, B, "C, D"
   * A - B
   * x: A - y: B
   * A - B, C - D, "E - F"
   * x: A - y: B,  x: C - y: D,  "x: E - y: F"
   * See field_handlers.feature for examples of usage.